draft-ajitomi-cose-cose-key-jwk-hpke-kem.md

```diff
@@ -153,7 +132,6 @@ A new generic kty(1) value HPKE-KEM(T.B.D.) is defined to represent the private
 A key with this kty has the following parameters:
 
 - The parameter kty(1) MUST be HPKE-KEM(T.B.D).
-- The parameter hkc(T.B.D.) MUST be present and contains the HPKE Key Configuration defined in Section 3.2.1.
```

Comment:
Is there any need to remove the entire line? Would it not be sufficient to change MUST to SHOULD or MAY?